exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -117,9 +117,13 @@ object SchemaUtils {
         }
     }
 
-    fun createFKey(reference: Column<*>): List<String> {
-        val foreignKey = reference.foreignKey
-        require(foreignKey != null && (foreignKey.deleteRule != null || foreignKey.updateRule != null)) { "$reference does not reference anything" }
+    fun createFKey(foreignKey: ForeignKeyConstraint): List<String> {
```

Comment:
Please keep old function (and call new one from it) with @Deprecated with warn level and ReplaceWith.
I do not expect a lot of direct usage of it but run it though deprecation flow is nice anyway.

Reply:
Done